README.md

```diff
@@ -43,3 +43,17 @@ reusable solutions and identify the capabilities that are missing to reach our g
 reproducible research workflows, by anyone, anywhere and anytime.
 
 [Go to the documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+
+## How to contribute
+We greatly appreciate and encourage contributions by any users or developers of 
+workflow tools. 
+A list of preselected tools can be found in the [documentation](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/).
+If you would like to add an implementation of the [simple use case](https://nfdi4ingscientificworkflowrequirements.readthedocs.io/en/latest/docs/simpleusecase.html) for a tool
+(not limited to the given list), or if you suggest changes to one of the
+existing implementations, please raise an issue and/or submit a pull request.
+We also encourage to start a new [discussion](https://github.com/BAMresearch/NFDI4IngScientificWorkflowRequirements/discussions)
+for general comments, questions and ideas you would like to share.
+
+## Acknowledgements
+We would like to thank the following people for their contribution to this repository:
+* TBA
```

Comment:
either remove this or add our names (but I guess this is already documented within github)

Reply:
I added this section to list people that might contribute by reviewing our implementations (i.e. the developers of the tools)
and for the (probably more rare case) that someone adds a implementation of the simple use case for some tool in the future

Reply:
@joergfunger I would keep this section and add later on the people contributing other implementations of the simple use case (kadi4mat / kadi studio, gnu guix).

Reply:
but isn't that that also seen in github, if they make a pull request? This way, we mention all others, but our contribution is not mentioned at all.

Reply:
okay, I removed the section. I agree that every contribution is already documented within github. So no need to list manually.